code/script.js

```diff
@@ -1,47 +1,317 @@
-// All the DOM selectors stored as short variables
+//Global variables
+
 const chat = document.getElementById('chat')
+const sendBtn = document.querySelector(".send-btn");
+const inputWrapper = document.querySelector(".input-wrapper");
+let defaultForm = document.getElementById("name-form");
+
+//Global variables - to store address and time bookig from user
+let address = "";
+let time = "";
+
+
+//Music
+const pop = new Audio("pop.flac");
 
-// Global variables, if you need any, declared here
+//Video control
+const videoBtn = document.querySelector(".video-btn");
+const video = document.querySelector(".video")
+
+videoBtn.addEventListener("click", function(){
```

Comment:
As a suggestion, here you could use the arrow function (ES6) instead. Also there's a gap missing between the parenthesis and the curly bracket on row 20 :)